Microsoft.Toolkit/Microsoft.Toolkit.csproj

```diff
@@ -14,14 +14,14 @@
     <PackageTags>Windows;Community;Toolkit;WCT;UWP;Incremental;Loading;Collection;IncrementalLoadingCollection;String;Array;extensions;helpers</PackageTags>
   </PropertyGroup>
 
-  <!-- .NET Standard 1.4 doesn't have the [Pure] attribute -->
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.4' ">
-    <PackageReference Include="System.Diagnostics.Contracts" Version="4.3.0" />
-  </ItemGroup>
-
   <!-- .NET Standard 2.1 and .NET 5 already have [NotNullIfNotNull] and [NotNullWhen] -->
-  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.1' OR '$(TargetFramework)' == 'net5.0' ">
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.1' OR '$(TargetFramework)' == 'net5.0'">
     <DefineConstants>NETSTANDARD2_1_OR_GREATER</DefineConstants>
   </PropertyGroup>
 
+  <!-- .NET Standard 1.4 doesn't have the [Pure] attribute -->
```

Comment:
Is there a reason this was moved down? I believe we originally trying to structure these to have oldest to newest framework conditions? Eh @Sergio0694?

Reply:
Having properties first and then items is just a style preference. But I observed that having these sections ordered this way improves readability of the project file.

Reply:
@michael-hawker Sorry for the late reply - yeah that's correct, we sorted multi-target sections from oldest to newest, so that it was easier to track additional NuGet references for them as they kept stacking over each other 👍